include/Console/console.h

```diff
@@ -263,6 +264,36 @@ namespace Hop
             return;
         }
 
+        void loadPackedScripts(std::string file)
+        {
+            scripts.load(file);
+        }
+
+        int require(lua_State * lua)
```

Comment:
to be tested

Reply:
there is now a test in ctest, works, require now part of Scriptz which is in extraspace